sklearn/gaussian_process/gpc.py

```diff
@@ -595,8 +596,8 @@ def fit(self, X, y):
         self.n_classes_ = self.classes_.size
         if self.n_classes_ == 1:
             raise ValueError("GaussianProcessClassifier requires 2 or more "
-                             "distinct classes. Only class %s present."
```

Comment:
The previous message *is* more informative...

Reply:
right, I added the present class again.